frame/support/src/lib.rs

```diff
@@ -114,6 +115,15 @@ impl TypeId for PalletId {
 	const TYPE_ID: [u8; 4] = *b"modl";
 }
 
+/// A parent blockchain identifier. Used primarily when parachains need to refer to a parent chain's
+/// `AccountId`.
+#[derive(Clone, Eq, PartialEq, Encode, Decode, TypeInfo)]
+pub struct ParentChainId(pub BoundedVec<u8, ConstU32<20>>);
```

Comment:
Do we really need this in Substrate? Why not have it in Polkadot?

Reply:
Hmm yeah, perhaps it makes more sense to put it in polkadot primitives.